tests/test_lib_crypto.py

```diff
@@ -36,15 +36,16 @@ def test_00_security_module_base_class(self):
 
         self.assertRaises(NotImplementedError, hsm.setup_module, {})
         self.assertRaises(NotImplementedError, hsm.random, 20)
-        self.assertRaises(NotImplementedError, hsm.encrypt, "20")
-        self.assertRaises(NotImplementedError, hsm.decrypt, "20")
+        self.assertRaises(NotImplementedError, hsm.encrypt, "20", 'abcd')
```

Comment:
Huh, what's this :-)

Reply:
I added the `iv` as a default parameter to the base class functions. Before it was set to `iv=None`.

Reply:
Ah my bad, didn't notice. Sounds good!